silverback/runner.py

```diff
@@ -55,6 +100,23 @@ async def run(self):
         Raises:
             :class:`~silverback.exceptions.Halt`: If there are no configured tasks to execute.
         """
+        if settings.MONGODB_URI:
+            self.persistence = await init_mongo(settings.MONGODB_URI)
```

Comment:
Does this make sense?  Do we really want to feed strings for module import?  If so, how do we handle settings like `MONGODB_URI`?